### DISCUSSION TITLE
ip已被封禁如何解决

### DISCUSSION THREAD

**Discussion by DongdongGui** [lyswhut/lx-music-desktop#2181] [Q&A]
ip已被封禁，禁止批量下载，请规范使用，此音乐源完全免费，若想解封请访问github上发布这个音源的项目查看解封方法，若您不知道这个项目，请在github上搜索洛雪音乐音源。

请问该问题如何解决

**Comment by Joker9267** [comment]
同问都没有下载过歌就说我批量下载，服了

**Comment by baba245** [comment]
清除一下缓存

**Comment by SandmanFMJ** [comment]
有些源设定的很极限，你切歌快了多了都会给你封了，换个宽松点的源去播放吧
